### DISCUSSION TITLE
Search by nested relationships

### DISCUSSION THREAD

**Discussion by KarlaRomero17** [tailflow/laravel-orion#281] [Help]
Hello, i'm new with orion and i encountered an issue on my code. 
These are the tables:
 - Location
    --City
       ---Country

i want to search by the name of the country and i got this error:
`
"message": "Call to undefined method location.city.country()",
`

The functions are:
```
protected function alwaysIncludes() : array
{
    return ['location.city.country'];
}

protected function searchableBy() : array
{
    return ['location.city.country.name'];
} 
```

I checked the relationships on models and it looks apparently formed correctly.

if you any idea, please let me know. Thanks

**Comment by v-radev** [comment]
Hey @KarlaRomero17, is this your location controller? Because in that case you should have just `['city.country']`. Also have you tried loading just `location` or `location.city` to see if that works?

**Comment by Giorgio-24** [comment]
Hi, I have the same problem. I solved it by downgrading from version 2.22.5 to version 2.20.0, which is the first version I've found where this bug is not present, this is what i was trying to do


```
    public function searchableBy(): array
    {
        return ['closed_at', 'appointment.pharmacy.name'];
    }
```

And this is the error message:

`"message": "Call to undefined method Report::appointment.pharmacy()",`
